### DISCUSSION TITLE
do python tests work?

### DISCUSSION THREAD

**Discussion by romange** [taskforcesh/bullmq#1967] [Q&A]
when running python tests locally like this:
`python3 -m unittest -v tests.delay_tests`

I get `user_script:39: Script attempted to access nonexistent global variable 'getListItems'`.
When further inspecting the lua script body I see the following:
```
-- Includes
--- @include "includes/addDelayMarkerIfNeeded"
--- @include "includes/addJobWithPriority"
--- @include "includes/getTargetQueueList"
--- @include "includes/trimEvents"
--- @include "includes/getNextDelayedTimestamp"
--- @include "includes/updateParentDepsIfNeeded"
```
is it something that should be expanded by bullmq python framework?

**Comment by manast** [comment]
The Python package uses the same lua scripts as the node version. Currently, we have a special include functionality for .lua scripts that needs to be compiled using "yarn build" which is part of the BullMQ package for node.

**Reply by manast** [reply]
In other words, you need to have Node and then use yarn or npm to run the build script to generate the final .lua scripts that can be used by the python package.

**Reply by romange** [reply]
Got ya. So yarn build. Where the compiled scripts reside? Are they under the repo directory?

**Reply by manast** [reply]
Yes, they are placed under rawScripts
